Small tweak to the header added by Debian's fixes/absl-optional.patch to
allow it to compile with jammy's libstdc++-11.

--- a/optional
+++ b/optional
@@ -42,7 +42,7 @@
 #include <bits/exception_defines.h>
 #include <bits/functional_hash.h>
 #include <bits/stl_construct.h> // _Construct
-#include <bits/utility.h> // in_place_t
+#include <utility> // in_place_t
 #if __cplusplus > 201703L
 # include <compare>
 # include <bits/invoke.h> // std::__invoke
